app/controllers/users_controller.rb

```diff
@@ -45,7 +45,7 @@ def destroy
     end
 
     def pending
-      users = User.where(approved: false)
+      users = User.where(approved: false).order(:emp_id).reverse
       head :no_content if users.empty?
```

Comment:
this statement is not needed.
let the render json: return empty hash if not users are found.